content/metadata.yaml

```diff
@@ -26,6 +26,17 @@ author_info:
     email: slochower@gmail.com
     affiliations:
       - Skaggs School of Pharmacy and Pharmaceutical Sciences, University of California, San Diego
+  -
+    github: vsmalladi
+    name: Venkat S. Malladi
+    initials: VSM
+    orcid: 0000-0002-0144-0564
+    twitter: katatonikat
+    email: vsmalladi@gmail.com
+    affliations:
+      - Department of Bioinformatics, University of Texas Southwestern Medical Center, Dallas, Texas, USA
+      - Bioinformatics Core Facility, University of Texas Southwestern Medicl Center, Dallas, Texas, USA
```

Comment:
There is a typo in `Medicl`, and you can remove the city, state, and country for now.